src/views/Restoration.vue

```diff
@@ -1,12 +1,21 @@
 <template>
-  <section class="pb-10" id="restoration-view">
+  <section class="pb-10" id="question_container">
     <!-- Company Information page-->
     <v-slide-x-transition hide-on-leave>
       <div v-if="!isSummaryMode">
         <header>
           <h1>{{ entityTitle }}</h1>
         </header>
 
+        <QuestionWrapper v-if="isLimitedConversionRestorationFiling"
+          id="heading-zero"
+          title="Applicant Relationship"
+          subtitle="Please select applicant's relationship to the company at the time the company was dissolved">
+          <RelationshipsPanel
+            class="ml-4 pl-5 pt-1"
+          />
+        </QuestionWrapper>
+
```

Comment:
Will 'wire-up' validation and setter events when the other tickets that complete this form have been completed.